sklearn/decomposition/kernel_pca.py

```diff
@@ -81,6 +81,9 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         A pseudo random number generator used for the initialization of the
         residuals when eigen_solver == 'arpack'.
 
+    copy : boolean, default=True
+        If True, input X is copied and stored by the model.
```

Comment:
I would actually write: "If no further changes will be done to X, setting ``copy=False`` saves memory by storing a reference."

Just to make it clear why this argument is useful.